docs/src/use/configure/configuration-files.md

```diff
@@ -423,6 +423,10 @@ export default defineConfig([
 
 #### Cascading Configuration Objects
 
+::: tip
+Cascading provides automatic configuration layering, while [`extends`](#extending-configurations) offers explicit inheritance. Choose cascading for progressive file-based configuration and extends for reusing established configurations.
```

Comment:
Add a link to the new section as well?

Also I think introducing the word "layering" is hard here. What does that mean?

Reply:
Yes a link would be helpful.

You can use 'merging' if 'layering' seems hard?